app/frontend/package.json

```diff
@@ -24,7 +24,7 @@
     "leaflet-edgebuffer": "^1.0.6",
     "leaflet-fullscreen": "^1.0.2",
     "leaflet-geosearch": "^2.7.0",
-    "leaflet-gesture-handling": "^1.1.8",
+    "leaflet-gesture-handling": "mrleblanc101/Leaflet.GestureHandling#c78f7414c8615a56d4eab54f4be5b9b34149afec",
```

Comment:
Would this create a technical debt in the future given it's not the official version?

Reply:
It is a possibility. But the main project project has been 1.1.8 for over a year. There is no signs in the issue tracker that the project owner is updating it anymore either. The bug is severe enough that we should either remove the plugin completely or live with this forked version IMO.